src/test/java/com/notably/commons/core/Path/RelativePathTest.java

```diff
@@ -0,0 +1,41 @@
+package com.notably.commons.core.path;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+
+/**
+ * Test for RelativePath.
+ */
+class RelativePathTest {
+
+    @Test
+    public void createRelativePath() {
+        assertThrows(InvalidPathException.class, () -> RelativePath.fromString("/CS2103/notes"));
+        try {
+            assertEquals(new RelativePath("CS2103/notes"), RelativePath.fromString("CS2103/notes"));
+        } catch (InvalidPathException ex) {
+            System.out.println("Error");
+        }
+
+    }
+
+
+    @Test
+    public void convertRelativePath() {
```

Comment:
Follow the test case method's naming convention [here](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html)!